edx/edx-ora2
# [BD-05][BB-3842] Fix issue with ORA pluggable submission editor

This PR fixes ORA pluggable submission editor issues in the staff area.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-3842

~~**Discussions**:~~
**Dependencies**: None

**Screenshots**: 
Staff area -

![localhost_18000_courses_course-v1_edX+DemoX+Demo_Course_courseware_9fca584977d04885bc911ea76a9ef29e_07bc32474380492cb34f76e5f9d9a135_2_activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%40ecd8e99c8a2741588ba455](https://user-images.githubusercontent.com/1010244/108879012-02f1bb80-762b-11eb-94f3-0e6dc516a910.png)

Instructor Dashboard -

![localhost_18000_courses_course-v1_edX+DemoX+Demo_Course_instructor](https://user-images.githubusercontent.com/1010244/108879563-9925e180-762b-11eb-9f3c-5f971e0ad02f.png)

Staff Grading with TinyMCE Response -

![localhost_18000_courses_course-v1_edX+DemoX+Demo_Course_courseware_9fca584977d04885bc911ea76a9ef29e_07bc32474380492cb34f76e5f9d9a135_3_activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%40f7c31e00e7474dd2bc (1)](https://user-images.githubusercontent.com/1010244/108886305-038e5000-7633-11eb-994c-e5650b103723.png)


**Sandbox URL**: TBD

**Merge deadline**: ASAP

**Testing instructions**:

1. Pull this PR in your local devstack and install using `make install-local-ora`.
2. In your `lms.yml` set the feature flag `ENABLE_XBLOCK_VIEW_ENDPOINT` to `true`.
3. Restart LMS
2. Go to the studio and create a new ORA Block with Staff grading.
3. Publish the ORA Block.
4. Go to the ORA block in LMS.
5. Make a submission.
6. Click on the `Manage Individual Learners` button.
7. Enter the username you've used to do the submission.
8. From the result click `Learner's Response`.
9. Make sure that the submission text has been loaded as expected.
10. Click on the `Grade Available Responses` button.
11. Click on the `Staff Assessment` link.
12. Make sure that the submission text has been loaded correctly in staff grading.
13. Go to the instructor dashboard.
14. Visit the `ORA Responses` tab.
15. Click on the ORA block you have created.
16. Make sure that the submission texts are showing correctly.

**Author notes and concerns**:
N/A

**Reviewers**
